src/lib/AppRegistry.tsx

```diff
@@ -49,21 +51,23 @@ class RootContainer extends React.Component<Props, {}> {
           this.state.retrying = true
           return <LoadFailureView onRetry={retry} style={{ flex: 1 }} />
         }}
-        {...untypedProps}
       />
     )
   }
 }
 
-class Artist extends RootContainer {
+class Artist extends React.Component<Props, {}> {
   constructor(props) {
     super(props)
     this.component = Containers.Artist
     this.route = new Routes.Artist({ artistID: props.artistID })
   }
```

Comment:
I know, but personally, I quite like it like this. Are there performance implications from setting up the props like this? I feel like they belong to the Artist / Gene / etc component, and then all of them use and render the `RootContainer` which holds those props, but consistently refer to `this.component` etc. It makes the `<RootContainer props />` bit less cluttered :o.